src/api/index.js

```diff
@@ -91,6 +91,13 @@ export async function getCurrentUser() {
   return currentUser;
 }
 
+export function getUserFromLocalStorage() {
```

Comment:
Even the `getCurrentUser` function will return the last cached value of the current user when it is not undefined. Probably I can directly use the global `currentUser` variable.

Reply:
I meant that in my point of view, should be only one method to get the user which is `getCurrentUser`. You, as consumer, don't need to know how it implemented. Say, one day we'll replace the key in the `localStorage`, we'll have to do this both in `getCurrentUser` and `getUserFromLocalStorage`.
Accessing to `currentUser` is also not recommended. `currentUser` is sort of `private` variable to avoid later modification.
In the bottom line, you have an API that returns a `Promise` resolved with user details, you can use it for free 😀